sdk/iot/Azure.Iot.Hub.Service/src/IoTHubServiceClient.cs

```diff
@@ -54,24 +61,91 @@ protected IoTHubServiceClient()
         /// <summary>
         /// Initializes a new instance of the <see cref="IoTHubServiceClient"/> class.
         /// </summary>
-        public IoTHubServiceClient(Uri endpoint)
-            : this(endpoint, new IoTHubServiceClientOptions())
+        /// <param name="connectionString">
+        /// The IoT Hub connection string, with either "iothubowner", "service", "registryRead" or "registryReadWrite" policy, as applicable.
+        /// For more information, see <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-security#access-control-and-permissions"/>.
```

Comment:
I'm not sure how this will show up. It might be a bare URL. You might consider giving it an end tag, and name the link.

Reply:
it shows up like this in the tooltip: 
![image](https://user-images.githubusercontent.com/22563986/84556123-91a8ed00-acd5-11ea-9937-1cf886966d20.png)

Reply:
Looks pretty good, I suppose. :)